README.md

```diff
@@ -4,3 +4,25 @@ Deploying an end-to-end observability system comes with many challenges. The org
 
 Schedule: https://kccnceu2023.sched.com/event/1HyZ3/
 Slides: https://docs.google.com/presentation/d/1oDpQo9KW_C5HznE0GR53P22HzP_SN2Ml/edit#slide=id.g227fe4440bd_0_0
+
+---
+
+# kubecon-eu-2023-opentelemetry-kubernetes-tutorial
+
+## 1. Setup observability backend
+
+After installing the observability backend, metrics, logs and traces can be sent via gRPC to `otel-collector.observability-backend.svc.cluster.local:4317` in otlp format.
+```bash
+$ kubectl apply -f https://raw.githubusercontent.com/pavolloffay/kubecon-eu-2023-opentelemetry-kubernetes-tutorial/main/k8s/backend.yaml
```

Comment:
remove the `$` or add it to all code snippets (I would prefer to remove it - it's then easier to copy/paste)